src/components/NewWorkspaceModal.js

```diff
@@ -179,6 +194,14 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
   const sourceLocationType = getLocationType(sourceWorkspaceLocation)
   const destLocationType = getLocationType(bucketLocation)
 
+  const onFocusAria = ({ focused, isDisabled }) => {
+    return `You are currently focused on ${isDisabled ? 'disabled ' : ''}option ${focused['aria-label']}.`
```

Comment:
Could we change this to put the option first so that it's more like the default? Hearing what you have selected first seems more convenient to me
```suggestion
    return `${isDisabled ? 'disabled ' : ''}option ${focused['aria-label']} focused.`
```

Reply:
@marctalbott I had it that way first, but I found that it didn't work well with the message that explains _why_ it is disabled. Given that whenever an element is disabled it will also have the message, this seemed to me to be much clearer.

Reply:
Or perhaps you are commenting on specifically removing "You are currently focused on"? With respect to that, I found it to be much easier to understand with the prefix because the billing project names can be pretty confusing (since they can't have spaces and the like). Starting with the prefix gave me context that was really helpful.

Reply:
The main thing for me was that "You are currently focused on" made it much slower to navigate the menu with a screen reader because I had to wait for the same repeated message to be read before I knew what the next option in the list was. Putting the selected option first would alleviate that, although I didn't test with many inaccessible billing projects so I didn't run into the pain that you're describing

Reply:
I'll play around with it some more and see about shortening the message. I agree the amount of text is crazy long, which is annoying because then it can pile up with text that needs to be read out. I imagine though that there is a keystroke to cut off the text… I just don't know what it is.

Reply:
@marctalbott pushed up the shorter message. I have grown to like it already. :)